<ui version="4.0" >
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>156</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>About Vidalia</string>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lblVidaliaLogo" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>132</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../res/vidalia.qrc" >:/images/128x128/vidalia-logo.png</pixmap>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="1" column="0" >
      <widget class="QLabel" name="lblVidalia" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Vidalia</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="lblVidaliaVersion" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>0.2.0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="lblTor" >
       <property name="text" >
        <string>Tor</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="lblTorVersion" >
       <property name="font" >
        <font>
         <italic>false</italic>
        </font>
       </property>
       <property name="text" >
        <string>0.2.0.31-alpha-dev (r123456)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="lblQt" >
       <property name="text" >
        <string>Qt</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLabel" name="lblQtVersion" >
       <property name="font" >
        <font>
         <italic>false</italic>
        </font>
       </property>
       <property name="text" >
        <string>4.4.2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QPushButton" name="btnShowLicense" >
         <property name="text" >
          <string>License</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0" >
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/vidalia.qrc" />
 </resources>
 <connections/>
</ui>
